.github/workflows/build.yml

```diff
@@ -8,34 +11,27 @@ on:
     branches: [ master ]
 
 jobs:
-  Windows:
-    runs-on: windows-latest
+  Build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+    runs-on: ${{ matrix.os }}
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Setup .NET
-      uses: actions/setup-dotnet@v1
+      uses: actions/setup-dotnet@v3
       with:
         dotnet-version: 6.0.402
     - name: Build
-      run: .\build.cmd -t Bundle
-      
-  Linux:
+      run: ./build.sh -t Bundle
+      shell: bash
```

Comment:
How does this work on Windows?

Reply:
Seems to be catered for (didnt delve into https://github.com/actions/runner/issues/497 in depth...)

Reply:
Alright, seeing is believing.